unstructured/ingest/evaluate.py

```diff
@@ -165,80 +81,16 @@ def measure_text_edit_distance(
     source_list: Optional[List[str]],
     export_dir: str,
 ):
-    """
-    Loops through the list of structured output from all of `output_dir` or selected files from
-    `output_list`, and compare with gold-standard of the same file name under `source_dir` or
-    selected files from `source_list`.
-
-    Calculates element type frequency accuracy and percent missing. After looped through the
-    whole list, write to tsv. Also calculates the aggregated accuracy.
-    """
-    if not output_list:
-        output_list = _listdir_recursive(output_dir)
-    if not source_list:
-        source_list = _listdir_recursive(source_dir)
-
-    rows = []
-    accuracy_scores: List[float] = []
-
-    for doc in output_list:  # type: ignore
-        fn = (doc.split("/")[-1]).split(".json")[0]
-        doctype = fn.rsplit(".", 1)[-1]
-        connector = doc.split("/")[0]
-        if doc in source_list:  # type: ignore
-            output = get_element_type_frequency(_read_text(os.path.join(output_dir, doc)))
-            source = get_element_type_frequency(_read_text(os.path.join(source_dir, doc)))
-            accuracy = round(calculate_element_type_percent_match(output, source), 3)
-            rows.append([fn, doctype, connector, accuracy])
-            accuracy_scores.append(accuracy)
-
-    headers = ["filename", "doctype", "connector", "element-type-accuracy"]
-    _write_to_file(export_dir, "all-docs-element-type-frequency.tsv", rows, headers)
-
-    agg_rows = []
-    agg_rows.append(
-        [
-            "element-type-accuracy",
-            _mean(accuracy_scores),
-            _stdev(accuracy_scores),
-            _pstdev(accuracy_scores),
-            len(accuracy_scores),
-        ],
+    return measure_element_type_accuracy(
+        output_dir, output_list, source_dir, source_list, export_dir
     )
-    _write_to_file(export_dir, "aggregate-scores-element-type.tsv", agg_rows, agg_headers)
-    _display(agg_rows, agg_headers)
-
-
-def _listdir_recursive(dir: str):
-    listdir = []
-    for dirpath, _, filenames in os.walk(dir):
-        for filename in filenames:
-            # Remove the starting directory from the path to show the relative path
-            relative_path = os.path.relpath(dirpath, dir)
-            if relative_path == ".":
-                listdir.append(filename)
-            else:
-                listdir.append(f"{relative_path}/{filename}")
-    return listdir
-
-
-def _write_to_file(dir: str, filename: str, rows: List[Any], headers: List[Any], mode: str = "w"):
-    if mode not in ["w", "a"]:
-        raise ValueError("Mode not supported. Mode must be one of [w, a].")
-    if dir and not os.path.exists(dir):
-        os.makedirs(dir)
-    with open(os.path.join(os.path.join(dir, filename)), mode, newline="") as tsv:
-        writer = csv.writer(tsv, delimiter="\t")
-        if mode == "w":
-            writer.writerow(headers)
-        writer.writerows(rows)
 
 
 def _display(rows, headers):
```

Comment:
I left _display here on the ingest side since it uses click.echo, but I get a vscode warning that the function is unused (in this file). Happy to move it to /metrics along with everything else but thought it would be cleaner to have all click-related functions in one file. Let me know!

Reply:
I agree on the formatting. Can we ignore the error? Mot sure which command to use but can try to ##ignore and something like unused?

Reply:
i don't believe its actually an error (but we'll see what ingest says), it's just greyed out on vscode. might not need to mark anything